typings/tooltip.d.ts

```diff
@@ -0,0 +1,29 @@
+/* tslint:disable */
+declare module "rc-tooltip" {
```

Comment:
It might be working for others..... on my side it fails i think when bundling...cause it doesn't pick the default method or class...